packages/plugin/src/sibling-operations.ts

```diff
@@ -18,6 +18,20 @@ import { dirname, join } from 'path';
 export type FragmentSource = { filePath: string; document: FragmentDefinitionNode };
 export type OperationSource = { filePath: string; document: OperationDefinitionNode };
 
+export const operationsLoaders = (): Loader<string, SingleFileOptions>[] => [
```

Comment:
I'm not sure this must be a function that re-creates the instance every time. ESLint is creating the parser services per each file, so it might create tons of instances of each loader.

Reply:
Ok, I don't know if the loaders instances are reusable or should be recreated each time. I will look at the code to see if it can be a problem.

Reply:
It's reusable, for sure, and it's completely stateless.